src/plugins/auto/src/infer_request.cpp

```diff
@@ -105,7 +105,7 @@ std::vector<ov::ProfilingInfo> ov::auto_plugin::InferRequest::get_profiling_info
     if (m_shared_request)
         return m_shared_request->get_profiling_info();
     if (m_scheduled_request)
-        return m_shared_request->get_profiling_info();
+        return m_scheduled_request->get_profiling_info();
```

Comment:
btw, the related tests is not enabled in CI, a following PR for this
https://github.com/openvinotoolkit/openvino/pull/18405/